politeiawww/cmd/politeiawwwcli/commands/cmsmanageuser.go

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package commands
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+)
+
+// CMSManageUserCmd allows an administrator to update Domain, ContractorType
+// and SupervisorID of a given user.
+type CMSManageUserCmd struct {
+	UserID             string `long:"userid" optional:"true" description:"User ID of the user to edit information"`
```

Comment:
I actually wanted to talk to you about this.  I write a lot of bash scripts to help automate testing PRs and forcing the prompts makes this difficult.  For new commands, what about doing something like either:

1. Only use the prompts if the user input doesn't contain enough info for a valid request.  For example, if the user didn't include any of the optional flags for the `CMSManageUserCmd` then it would go through the prompts that you have there already.  If the user does use one or more of the optional flags then its a valid request and gets sent as is without prompting the user for any more input.

or

2. Take the prompts out completely.

What do you think?

Reply:
Ah gotcha, yah sounds totally reasonable to go with option 1.  I'll make those changes to this PR and keep that in mind for future commands.   I'm not a huge fan of the prompts anyways.